spec/features/edit_me_spec.rb

```diff
@@ -0,0 +1,11 @@
+describe "User can manage their own profile" do
```

Comment:
I was assuming we'd be using RESTful routes and this action would be `profiles/:id/edit` -- if we are sticking with `/me` then it would not be testable